<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PreferencesForm</class>
 <widget class="QWidget" name="PreferencesForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Préférences</string>
  </property>
  <widget class="QPushButton" name="bt_apply">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>260</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Apply</string>
   </property>
  </widget>
  <widget class="QPushButton" name="bt_ok">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>260</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>381</width>
     <height>241</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Palatino Linotype</family>
    </font>
   </property>
   <property name="currentIndex">
    <number>2</number>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>General</string>
    </attribute>
    <widget class="QGroupBox" name="groupBox">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>10</y>
       <width>341</width>
       <height>111</height>
      </rect>
     </property>
     <property name="title">
      <string>Système</string>
     </property>
     <widget class="QCheckBox" name="chkbx_launchOnStart">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>301</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text">
       <string>start Zemini on your computer boot</string>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>130</y>
       <width>341</width>
       <height>71</height>
      </rect>
     </property>
     <property name="title">
      <string>Language</string>
     </property>
     <widget class="QComboBox" name="cb_language">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>321</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>Account</string>
    </attribute>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>351</width>
       <height>81</height>
      </rect>
     </property>
     <property name="title">
      <string>Informations </string>
     </property>
     <widget class="QLabel" name="label">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>46</width>
        <height>13</height>
       </rect>
      </property>
      <property name="text">
       <string>Email :</string>
      </property>
     </widget>
     <widget class="QLabel" name="lb_otherLocation">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>111</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Zemini folder location :</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="le_email">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>20</y>
        <width>141</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="le_otherLocation">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>50</y>
        <width>191</width>
        <height>20</height>
       </rect>
      </property>
      <property name="accessibleDescription">
       <string extracomment="ij"/>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_5">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>100</y>
       <width>351</width>
       <height>80</height>
      </rect>
     </property>
     <property name="title">
      <string>Directory Settings</string>
     </property>
     <widget class="QLabel" name="label_9">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>171</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Number of thumbnails by video : </string>
      </property>
     </widget>
     <widget class="QComboBox" name="cb_nbThumbnails">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>30</y>
        <width>69</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_3">
    <attribute name="title">
     <string>Statistics</string>
    </attribute>
    <widget class="QProgressBar" name="progressBar1">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>20</y>
       <width>201</width>
       <height>23</height>
      </rect>
     </property>
     <property name="value">
      <number>24</number>
     </property>
    </widget>
    <widget class="QLabel" name="progressBarLabel1">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>category1</string>
     </property>
    </widget>
    <widget class="QLabel" name="progressBarLabel2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>60</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>category2</string>
     </property>
    </widget>
    <widget class="QProgressBar" name="progressBar2">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>60</y>
       <width>201</width>
       <height>23</height>
      </rect>
     </property>
     <property name="value">
      <number>24</number>
     </property>
    </widget>
    <widget class="QLabel" name="progressBarLabel3">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>100</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>category3</string>
     </property>
    </widget>
    <widget class="QProgressBar" name="progressBar3">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>100</y>
       <width>201</width>
       <height>23</height>
      </rect>
     </property>
     <property name="value">
      <number>24</number>
     </property>
    </widget>
    <widget class="QLabel" name="progressBarLabel5">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>180</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>category5</string>
     </property>
    </widget>
    <widget class="QProgressBar" name="progressBar5">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>180</y>
       <width>201</width>
       <height>23</height>
      </rect>
     </property>
     <property name="value">
      <number>24</number>
     </property>
    </widget>
    <widget class="QProgressBar" name="progressBar4">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>140</y>
       <width>201</width>
       <height>23</height>
      </rect>
     </property>
     <property name="value">
      <number>24</number>
     </property>
    </widget>
    <widget class="QLabel" name="progressBarLabel4">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>140</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>category4</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_4">
    <attribute name="title">
     <string>Sync</string>
    </attribute>
    <widget class="QGroupBox" name="groupBox_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>351</width>
       <height>101</height>
      </rect>
     </property>
     <property name="title">
      <string>Network Sync settings</string>
     </property>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>30</y>
        <width>101</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Pull in Data every :</string>
      </property>
     </widget>
     <widget class="QComboBox" name="cb_pullInOption">
      <property name="geometry">
       <rect>
        <x>258</x>
        <y>30</y>
        <width>81</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>70</y>
        <width>111</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Send out Data every :</string>
      </property>
     </widget>
     <widget class="QComboBox" name="cb_sendOutOption">
      <property name="geometry">
       <rect>
        <x>258</x>
        <y>70</y>
        <width>81</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QPushButton" name="bt_cancel">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>260</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
